spec/lib/wiki_pageviews_spec.rb

```diff
@@ -161,6 +161,8 @@
       let(:title) { 'Voyages,_aventures_et_combats/Chapitre_18' }
       let(:subject) { described_class.new(article).average_views }
 
+      before { travel_to Date.new(2023, 10, 18) }
+
```

Comment:
I had to add this because it looks like I visited the site yesterday so I broke the spec. 
I noticed that because the build for PR #5498 failed with:
```
  3) WikiPageviews.average_views_for_article for an article that exist but has no view data returns 0
     Failure/Error: expect(subject).to eq(0)

       expected: 0
            got: 1.0
```

This way it should be more robust.